### PR TITLE
Fix test case ReExecuteASTTest

### PR DESCRIPTION
**Pull request by ke-yu**
This recorded test case fails because it tries to use a node defeined in ``FFITarget.dll``, and recorded xml file doesn't contain a command to load this library. Adding a parameter to ``RunCommandsFromFile()`` so that we could specify libraries need to be pre-loaded. 

@Benglin  for your review.

### CODE REVIEW

**Comment by sharadkjaiswal** [comment]
Can we make use of DynamoUtilities.DynamoPathManager.AddPreloadLibrary() ??

**Comment by ke-yu** [comment]
@sharadkjaiswal , yep, you are correct, updated. :-)

**Comment by Benglin** [comment]
My memory must have failed me, I remember having other tests that make use of preloaded library. This LGTM by the way, feel free to merge.

File: test/DynamoCoreUITests/RecordedTests.cs [hunk 1]
```diff
@@ -1058,9 +1058,11 @@ public void TestUpdateNodeCaptions_DS()
             Assert.AreEqual("CBN", cbn.NickName);
         }
 
-        [Test, RequiresSTA, Category("Failure")]
+        [Test, RequiresSTA]
         public void ReExecuteASTTest()
         {
+            DynamoUtilities.DynamoPathManager.Instance.AddPreloadLibrary("FFITarget.dll");
```
Comment (sharadkjaiswal) [inline test/DynamoCoreUITests/RecordedTests.cs]: We might need an API to remove certain library from preload sequence, when library loader is re-initialized.

**Pull request merged**